content/docs/aws/deploy/existing-cluster.md

```diff
@@ -20,7 +20,7 @@ If you would like to deploy Kubeflow on existing Amazon EKS cluster, the only di
 
     # Download config files
     export CONFIG="/tmp/kfctl_aws.yaml"
-    wget https://raw.githubusercontent.com/kubeflow/kubeflow/master/bootstrap/config/kfctl_aws.yaml -O ${CONFIG}
+    wget https://raw.githubusercontent.com/kubeflow/kubeflow/v0.6-branch/bootstrap/config/kfctl_aws.yaml -O ${CONFIG}
```

Comment:
instead of hardcoding the versions in the url, we may be able to use hugo shortcodes that Sarah has setup for exactly this reason..

Reply:
That would be very helpful!
@sarahmaddox can you provide some detail on how to use them?

Reply:
Hmm, I think we may need an additional shortcode. The "latest-version" shortcode that we use for the docs specifies 3 levels of version. It's currently set to `v0.6.1`:
https://github.com/kubeflow/website/blob/master/layouts/shortcodes/kf-latest-version.html

But I notice that in this PR you need only `v0.6` - is that right?

Here's the syntax to use a shortcode:

```
{{% kf-latest-version %}}
```

And some [examples of its use](https://github.com/kubeflow/website/search?q=kf-latest-version&unscoped_q=kf-latest-version).

Reply:
Actually, I think we should use latest-version.
Having the full version allows for immutability of the config files, while v0.6-branch may change.
cc @jlewi